docs/apps/julia.md

```diff
@@ -0,0 +1,103 @@
+# Julia
+
+## Description
+
+Julia is a high-level, high-performance dynamic programming language for
+numerical computing. It provides a sophisticated compiler, distributed
+parallel execution, numerical accuracy, and an extensive mathematical
+function library.
+
+For a quick introduction and tutorial, see https://github.com/csc-training/julia-introduction 
+
+[TOC]
+
+## Available
+
+### Puhti
+-   1.1.1 compiled with Intel Math Kernel Library (MKL)
+-   1.1.1 from pre-compiled binaries
```

Comment:
Some more information for the users could be needed, i.e when should you use which version?

Reply:
I added recommendation to use pre-compiled binaries, when using ARPACK package.